scripts/install

```diff
@@ -191,6 +193,18 @@ form_secret=$(egrep "^form_secret" homeserver.yml | cut -d'"' -f2)
 ynh_app_setting_set $app registration_shared_secret "$registration_shared_secret"
 ynh_app_setting_set $app form_secret "$form_secret"
 
+# Create .well-known redirection for access by federation
+if [[ $server_name != $domain ]]
+then
+  cat > /etc/nginx/conf.d/${server_name}.d/server_name.conf <<EOF
+    location /.well-known/matrix/ {
+        return 200 '{"m.server": "$domain", "m.homeserver": "https://$domain"}';
+        add_header Content-Type application/json;
+        add_header Access-Control-Allow-Origin '*';
+        }
+EOF
```

Comment:
Well, I really prefer to use a template file than this

Reply:
OK. What is a template file?
I reach to this result after hours of researches, and tests.
I learnt a lot and I'm sure it can be improved. Let me know how.

Reply:
Well, just add the nginx config file content in a new file in `/conf`. And replace the string everywhere as same as for the other files.

Reply:
Well, I'm not sure to understand.
In `/etc/nginx/conf.d/${server_name}.conf`, it is mentioned to listen to 
`/etc/nginx/conf.d/${server_name}.d/*.conf`
so if we create a file in this directory it should be a clean process, no?
where is `/conf` folder?

Also, if I know how to create a file and put content in it now, modifying a file to replace strings... looks much more tricky and risky.
I'll remove the file in the remove script.

Reply:
Just do for this config file, as same as for the homeserver config file.

Reply:
Like that?

https://github.com/YunoHost-Apps/synapse_ynh/pull/136/commits/4f2de63065d28ac451c87d119798bdabcf1ef0dd

https://github.com/YunoHost-Apps/synapse_ynh/pull/136/commits/be8f0d25d68acf2be43702c2ff310f067323eccd